data.php

```diff
@@ -125,7 +125,7 @@ function getAllQueries() {
         $hostname = trim(file_get_contents("/etc/hostname"), "\x00..\x1F");
 
         foreach ($dns_queries as $query) {
-            $time = date_create(substr($query, 0, 16));
+            $time = exec("date --date='" . trim(substr($query, 0, 16)) . "' +'%Y-%m-%d\%Z%H:%M:%S'");
```

Comment:
sure! I thought the Timezone (`T` in php date) is used here and forgot about the function of the `\` ^^